toxcore/tox.api.h

```diff
@@ -742,6 +745,53 @@ bool bootstrap(string address, uint16_t port, const uint8_t[PUBLIC_KEY_SIZE] pub
 bool add_tcp_relay(string address, uint16_t port, const uint8_t[PUBLIC_KEY_SIZE] public_key)
     with error for bootstrap;
 
+namespace query {
+
+  /**
+   * Queries the server at IP port, with PUBKEY, for the TOXID at $name
```

Comment:
* PUBKEY is not a defined term, nor is TOXID.
* Add `.` at EOL.
* "IP port" is unclear.
* Actually I don't understand this sentence at all.
* `$name` is unresolved. Parameters are not renamed in apidsl, so no need to use references.